components/list/list.js

```diff
@@ -26,9 +25,7 @@ function List(props) {
 
   const notEmpty = i => i !== '' && i !== null;
 
-  const itemsToRender = (renderEmptyItems === false) ? items.filter(notEmpty) : items;
-
-  const itemsBlock = itemsToRender.map((item, index) => (
+  const itemsBlock = items.filter(notEmpty).map((item, index) => (
```

Comment:
Why can't we use `filter(Boolean)`?

Reply:
In that case, we will filter out `0` (as Number) values. It's probably the very rare case but still, there is a possibility to get it. And that will be a confusion I think.

Reply:
But we don't have `number` in `PropTypes` for the property

Reply:
That's a good point. 😄  Okay, it makes sense to change it then.